cloud/performance.rst

```diff
@@ -86,3 +86,30 @@ To measure the number of active connections at a given time, run:
   SELECT COUNT(*)
     FROM pg_stat_activity
    WHERE state <> 'idle';
+
+PostgreSQL/PgBouncer Configuration
+==================================
+
+In addition to adding nodes and scaling their hardware, the Cloud Console allows changing database server configuration parameters. Adjusting the parameters can help tune the server for particular workloads. Cloud currently exposes just those parameters which do not require a server restart to take effect.
+
+To access these settings, go to the Configuration tab in your Cloud formation and click "Change a Parameter." It will present a dropdown list of config parameters:
+
+.. image:: ../images/cloud-pg-conf.png
+
+The options are grouped by which server and system they control:
+
+* Coordinator PostgreSQL
+* Coordinator Inbound PgBouncer
+* Worker PostgreSQL
+
+Selecting an option opens an input box that accepts the appropriate values, whether numerical, textual, or a pre-set list. The selected option also shows a link to learn more about the configuration parameter.
+
+.. image:: ../images/cloud-pg-conf-option.png
+
+Numerical parameters do not yet allow specifying units in this interface, and are interpreted as their default unit. A reference of the default units is available in the database from the ``pg_settings`` table:
```

Comment:
The `short description` of tune settings mention the units most of the time, even if the actual documentation doesn't.